packages/wonder-blocks-dropdown/components/dropdown.test.js

```diff
@@ -0,0 +1,98 @@
+//@flow
+import React from "react";
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+
+import OptionItem from "./option-item.js";
+import Dropdown from "./dropdown.js";
+
+const keyCodes = {
+    tab: 9,
+    escape: 27,
+    space: 32,
+    up: 38,
+    down: 40,
+};
+
+describe("Dropdown", () => {
+    global.scrollTo = jest.fn();
+    global.getComputedStyle = jest.fn();
+
+    afterEach(() => {
+        unmountAll();
+    });
+
+    it("handles keyboard behavior as expected", () => {
```

Comment:
Given all of the different scenarios that dropdown.js handles I would expect more tests.  We probably want to cover the following scenarios in the test matrix:
- keyboard: false, initialFocusedIndex: 0
- keyboard: false, initialFocusedIndex: not 0
- keyboard: true, initialFocusedIndex: 0
- keyboard: true, initialFocusedIndex: not 0